lib/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @fileoverview Shareable eslint plugin for use by eslint-config-silvermine
+ * @author Justin Hummel
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var requireIndex = require("requireindex");
```

Comment:
We don't need to pull in another dependency just for the sake of saving a couple lines of `require('rules/some-rule-name');`.